keychain_linux.cpp

```diff
@@ -43,16 +45,32 @@ const SecretSchema makeSchema(const std::string &package) {
                         }};
 }
 
+std::string makeLabel(const std::string &service, const std::string &user) {
+    std::string label = service;
+
+    if (!user.empty()) {
+        label += " (" + user + ")";
+    }
+
+    return label;
+}
+
+void handleError(GError *error, Error &err) {
```

Comment:
Rename to `updateError` for consistency with other platforms
```suggestion
void updateError(GError *error, Error &err) {
```